core/client/src/main/java/alluxio/client/block/UnderStoreBlockInStream.java

```diff
@@ -148,6 +158,34 @@ public int read(byte[] b, int off, int len) throws IOException {
   }
 
   @Override
+  public int read(long pos, byte[] b, int off, int len) throws IOException {
+    Preconditions.checkState(PACKET_STREAMING_ENABLED,
+        "PositionedReadable interface is implemented only if packet streaming is enabled.");
```

Comment:
the error message is a bit weird and confusing here. Should it be  `"Positioned read is not supported, please set  " + USER_PACKET_STREAMING_ENABLED.toString() + "to true to enable positioned read"`